### ISSUE TITLE
[Spike] - Identify appointment service types as they correspond to service categories

### ISSUE THREAD

**Issue by cferris32** [department-of-veterans-affairs/va.gov-team#54558]
This research will be a follow up to this original spike ticket: https://app.zenhub.com/workspaces/vaos-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/52886

Now that we are able to track the service categories of appointments flowing through VAOS, we'd like to understand trends around what service types these appointments have when they are of certain service categories, namely C&P. This ticket will be for the logging and research to understand trends between service category and service type in staging and production.

Q1: What service types (if any) do we see for C&P appointments?
Q2: Are there other notable trends between these fields for other non-health appointments?

Timebox: 6 hours

**Comment by cferris32** [comment]
Datadog findings:

After a few days of logging, we've gathered some metrics around what serviceTypes we see for each serviceCategory. In general, we notice that serviceType is often populated with typical healthcare values (primaryCare, optometry, etc) even on non health appointments (meaning serviceCategory is not "REGULAR"). Here are counts from sample data for each serviceCategory:

We'll start with a sample count of health appointments (serviceCategory == REGULAR) to set a baseline. Here are a few different log groups captured for this category:
![Screenshot 2023-03-30 at 3.10.14 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/e38c8961-68fc-4649-aa42-80f456cb2a99)![Screenshot 2023-03-30 at 3.10.25 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/a7f1bbb5-03d6-4474-ba16-23b24abd9a6c)![Screenshot 2023-03-30 at 3.10.37 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/8600b692-7be8-46e6-a59f-6262d8f94c1d)
Now for sample metrics of the non health appointments, organized by serviceCategory. We'll start with our focus group, "COMPENSATION & PENSION", in which we see a trend of health related serviceTypes out of 66 entries:
![Screenshot 2023-03-30 at 4.00.54 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/9d6d4733-bf50-4a1f-8abf-96a5460d9976)
Next, "CLASS II DENTAL" (35 entries):
![Screenshot 2023-03-30 at 3.05.39 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/e978fe7c-f0a5-4b1c-87d3-aaecb0d4ea86)
Next, "COLLATERAL OF VET." (32 entries):
![Screenshot 2023-03-30 at 3.06.33 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/3c1aae9b-22cb-4afb-8e5b-6764d3166a4a)
"COMPUTER GENERATED" (4 entries):
![Screenshot 2023-03-30 at 3.07.03 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/aefe5002-d616-49d8-b98d-dcf33fac4c8d)
"EMPLOYEE" (18 entries and 23 entries, respectively):
![Screenshot 2023-03-30 at 3.07.42 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/68f44379-6222-4dba-bae0-cedec6287e17)![Screenshot 2023-03-30 at 3.07.52 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/eff7d6d1-cb28-4d39-b02c-eb6832d4a163)
"ORGAN DONORS" (of the 1K+ entries, all were of serviceType == socialWork, except for two):
![Screenshot 2023-03-30 at 3.08.55 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/c5f6589b-0778-4a65-a42e-dd091e4cc4ba)
"PRIMA FACIA" (281 entries):
![Screenshot 2023-03-30 at 3.09.28 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/f5b29a7c-e8ab-4461-bfbe-98d45a631680)
"RESEARCH" (82 entries):
![Screenshot 2023-03-30 at 3.11.04 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/939077e9-1a93-499d-bf1d-b01ab06109ef)
"SERVICE CONNECTED" shows a larger spread of types than most of our other non health categories, sitting closer to that of our health appointments:
![Screenshot 2023-03-30 at 3.11.37 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/a4bccbc8-5412-4709-9e80-b38853b88638)![Screenshot 2023-03-30 at 3.11.55 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/c37cb5e0-984d-4d9e-bdac-a2df377b1763)![Screenshot 2023-03-30 at 3.12.08 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/fa713cb0-dc90-4e5f-b4bc-7fd3b5e34c00)
We end with "SHARING AGREEMENT", which contained 23 entries of mostly optometry:
![Screenshot 2023-03-30 at 3.12.38 PM.png](https://images.zenhubusercontent.com/5fcfbc1c239ebd98b2acfe6c/df9eb8a1-b5a3-446c-ad35-a1c9afeb6e57)

In summary, we see a general trend of non health appointments continuing to make use of the health-related serviceType values. C&P appointments have a majority serviceType of "socialWork", and the service types of "socialWork" and "primaryCare" make up the most common values seen across different service categories, with only a couple exceptions such as serviceType == optometry for "SHARING AGREEMENT". With this discovery in mind, we might be able to begin inquiring about logic behind these type/category relationships.
